docs/FAQ.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: global
+title: Tachyon FAQ
+---
+
+## Tachyon Frequently Asked Questions.
+
+1. What is Tachyon?
+[Tachyon](http://tachyon-project.org/index.html) is an in-memory distributed, reliable storage system enabling data sharing across cluster at memory speed. Data reliability is achieved by using lineage concept. In memory speed is acheived by caching working file sets, avoiding performing disk reads.
+
+2. What platforms and Java versions does Tachyon run on?
+Currently Tachyon has been tested with Oracle Java 1.7 running on various distributions of Linux.
+
+3. Is there an easy way to see the status and health of a cluster?
+Yes, Once Tachyon is installed, please login into [http://master_ip:19999](http://master_ip:19999)
+
+4. If I add new nodes to the cluster, will tachyon rebalance/move the cached blocks to the newly added nodes in order to balance disk space utilization between the nodes?
+No, Currently re-balancing of data blocks in not available.
+
+5. How do you achieve Tachyon master redundancy?
+We use Tachyon Master/Stand-by architecture. Also, zookeeper can be used to pick a Tachyon master.
+
+6. What is underlayer FS and how many Underlayer FS are supported?
+Underlayer FS is a general distributed filesystems on which tachyon works seamlessly used as long term storage for fault tolerant and to re-compute missing data. Tachyon inherently is not a filesystem.
```

Comment:
Zookeeper is required for Master Fault Tolerance, yes it does pick the master but without it there is no fault tolerance